pennylane/qchem/dipole.py

```diff
@@ -251,7 +251,7 @@ def dipole_moment(mol, cutoff=1.0e-18, core=None, active=None):
 
     .. math::
 
-        \hat{D}_\mathrm{n} = \sum_{i=1}^{N_\mathrm{atoms}} Z_i {\bf R}_i \hat{I},
+        \hat{D}_\mathrm{n} = \sum_{i=1}^{N_\mathrm{atoms}} Z_i {\bf R}_i,
```

Comment:
Nice! It's good to take the opportunity in this PR to make small fixes throughout :brain: